docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Serial Vault User Guide
+table_of_contents: true
+---
+
+# Serial Vault user guide
+
+The Serial Vault is a Canonical-hosted web service that is required to use a
+[Secure Brand Store][secure-brand-store]. The Serial Vault provides :
+
+ - Signed [serial assertions][serial-assertions]  to authenticated systems of
+   a given brand/model, which then use it to access a secure Brand Store.
+ - Signed [system-user assertions][system-user-assertions], which allow
+   creating a system user account from a USB drive on an unmanaged Core system.
+   For additional information, see [here][here].
+
+After Canonical creates a Serial Vault account for you, you can login here:
+[https://serial-vault-partners-admin.canonical.com/]
+
+Serial Vault tasks include:
```

Comment:
"use cases" instead of "tasks"?

Reply:
Remove slash at the end of the URL; Perhaps rephrase to avoid repeating "you"

Reply:
no, these are actual tasks the user needs to perform in the SV.